.github/workflows/image-build.yaml

```diff
@@ -75,3 +75,38 @@ jobs:
       registry_username: ${{ secrets.QUAY_PUBLISH_ROBOT }}
       registry_password: ${{ secrets.QUAY_PUBLISH_TOKEN }}  
     
+  analyzer-windows-build:
+    runs-on: windows-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Login to Docker Hub
+      uses: docker/login-action@v3
+      with:
+        registry: "quay.io/konveyor"
+        username: ${{ secrets.QUAY_PUBLISH_ROBOT }}
+        password: ${{ secrets.QUAY_PUBLISH_TOKEN }}
+    - name: Docker Build
+      run: |
+        TAG=${GITHUB_REF_NAME/main/latest}
+        IMAGE_NAME=quay.io/konveyor/analyzer-lsp:${TAG}-ltsc2022
+        docker build -t ${IMAGE_NAME} -f ./Dockerfile.windows .
+        docker push ${IMAGE_NAME}
+
+  dotnet-windows-build:
+    runs-on: windows-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Login to Docker Hub
+      uses: docker/login-action@v3
+      with:
+        registry: "quay.io/konveyor"
+        username: ${{ secrets.QUAY_PUBLISH_ROBOT }}
+        password: ${{ secrets.QUAY_PUBLISH_TOKEN }}
+    - name: Docker Build
+      run: |
+        TAG=${GITHUB_REF_NAME/main/latest}
+        IMAGE_NAME=quay.io/konveyor/dotnet-external-provider:${TAG}-ltsc2022
+        docker build -t ${IMAGE_NAME} -f ./external-providers/dotnet-external-provider/Dockerfile.windows .
```

Comment:
can we add this to the anlalyzer-lsp manifest list instead?

Reply:
I am almost certain we can but the required rewrite of our reusable workflow ([build-push-images](https://github.com/konveyor/release-tools/blob/main/.github/workflows/build-push-images.yaml)) is too costly to be completed right now.

A couple of questions don't have immediately obvious answers:
1. How to deal with multiple Dockerfiles for the same image_name
1. Can you add images to an already pushed manifest?